client/landing/stepper/declarative-flow/internals/steps-repository/domains/domain-form-control.tsx

```diff
@@ -74,6 +74,7 @@ export function DomainFormControl( {
 	let showExampleSuggestions: boolean | undefined = undefined;
 	let includeWordPressDotCom: boolean | undefined = undefined;
 	let showSkipButton: boolean | undefined = undefined;
+	let shouldQuerySubdomains: boolean | undefined = undefined;
```

Comment:
```suggestion
	let shouldQuerySubdomains: boolean | undefined = true;
```
Should the default value here be `true`, like in the `RegisterDomainStep` component? Otherwise I think other `/setup/*` onboarding flows might not query subdomains.

Reply:
If it's `undefined`, then the `RegisterDomainStep.defaultProps` should take place as `true` - since these are used when the prop in question is `undefined`. But that's fine, I can change this one for extra caution.